docs/ref/files/file.txt

```diff
@@ -46,18 +46,24 @@ The ``File`` class
 
         The read/write mode for the file.
 
-    .. method:: open(mode=None)
+    .. method:: open(mode=None, *args, **kwargs)
 
         Open or reopen the file (which also does ``File.seek(0)``).
         The ``mode`` argument allows the same values
         as Python's built-in :func:`python:open()`.
+        ``*args`` and ``*kwargs`` are passed to
+        Python's built-in :func:`python:open()`.
```

Comment:
I wonder if we should say something like "are passed after mode" to be clear that mode is part of the signature, then additional args/kwargs, _except_ for mode, are passed.

Reply:
Yes, I think that's a good idea.